index.js

```diff
@@ -1,126 +1,19 @@
 module.exports = {
-  plugins: ['flowtype', 'import', 'prettier', 'promise', 'react', 'react-hooks'],
-  extends: ['plugin:flowtype/recommended', 'plugin:react/recommended', 'airbnb', 'prettier'],
+  plugins: [
+  ],
+  extends: [
+    'plugin:flowtype/recommended', 
+    'plugin:react/recommended', 
+    'airbnb', 
+    'prettier',
```

Comment:
This should belong to the `rules/prettier`, IMO

Reply:
For some reason extensions are not working if I move them to specific rule files, it seems the sequence is causing some rules to override (just a guess)

Reply:
IIRC the sequence indeed matters in terms of which rule takes precedence, but don't fully follow why moving the `prettier` config to the getsafe prettier subconfig doesn't work.